src/components/editMeasure/populationCriteria/PopulationCriteriaHome.test.tsx

```diff
@@ -212,4 +231,101 @@ describe("PopulationCriteriaHome", () => {
     );
     expect(screen.getByTestId("groupDescriptionInput")).toHaveTextContent("");
   });
+
+  it("should not render Reporting component for non-QDM measures", () => {
+    render(
+      <MemoryRouter
+        initialEntries={[
+          { pathname: "/measures/testMeasureId/edit/reporting" },
+        ]}
+      >
+        <ApiContextProvider value={serviceConfig}>
+          <Route path={["/measures/testMeasureId/edit/reporting"]}>
+            <PopulationCriteriaHome />
+          </Route>
+        </ApiContextProvider>
+      </MemoryRouter>
+    );
+    // verifies if the side nav is created and Reportin is not available
+    expect(
+      screen.queryByRole("tab", {
+        name: /Reporting/i,
+      })
+    ).toBeNull();
+  });
+
+  it("should not render base configuration component for non-QDM measures", () => {
+    render(
+      <MemoryRouter
+        initialEntries={[
+          { pathname: "/measures/testMeasureId/edit/base-configuration" },
+        ]}
+      >
+        <ApiContextProvider value={serviceConfig}>
+          <Route path={["/measures/testMeasureId/edit/base-configuration"]}>
+            <PopulationCriteriaHome />
+          </Route>
+        </ApiContextProvider>
+      </MemoryRouter>
+    );
+    // verifies if the side nav is created and Reportin is not available
+    expect(
+      screen.queryByRole("tab", {
+        name: /Base Configuration/i,
+      })
+    ).toBeNull();
+  });
+
+  // tests for QDM measures, since this test case uses a mock which returns QDM measure
+  // all further test cases will have QDMMeasure being returned from Measure store.
```

Comment:
What do you think about having a separate test file for QDM?

Reply:
Yeah, we can also do that, Honestly, I don't understand why jest test cases are carrying the config from one test case to consecutive test cases, especially when the test suite executes asynchronously.

I don't have a strong opinion either way, but this PopulationCriteriaHome page Is kinda small, so I feel having 2 different test files could be an overkill. 

Now that I think, we can have 2 different test suites 1 for QiCore and the other for QDM, wondering if the mocked modules can be differentiated. I will verify this.

Reply:
Scratch my idea of having 2 suites in the same file, that doesn't work for the mocking module at the component level.